containers/podman-desktop
# chore: switch to node16 modules instead of node modules in main package

### What does this PR do?
moving to something up-to-date :-)
will allow to bump got library

It was straightforward except for two libraries: `check-disk-usage` and `validator`. There are workarounds (we specify the function that is available manually)

For now, all imports in main package need to be suffixed by '.js' (you can browse typescript/esm docs)

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?



### How to test this PR?


But please check 'creating a new podman machine' (it should check the disk space using `check-disk-usage` library)

and try to add a valid registry URL and invalid registry URL (for `validator` library)



Change-Id: I854c62df5ba99d0bdc0b6f0f42bb653c7c990e8b
